.changeset/little-apricots-rhyme.md

```diff
@@ -1,5 +0,0 @@
----
-"@nextui-org/drawer": patch
----
-
-use top-level type-only import instead of inline type specifiers
```

Comment:
why deleting this changeset? please undo it.

Reply:
I made an incorrect commit.